### PR TITLE
Sparkly dichondra: Set Max-Width on Pop-overs

### PR DESCRIPTION
**Pull request by ttseng**
Should prevent very wide pop-overs such as 
image

Also removed extraneous padding on popover titles.

### CODE REVIEW

File: watch.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
     "include": ["^package\\.json$", "^\\.env$"]
   },
   "restart": {
-    "exclude": ["^lib/"],
```
Comment (sarahzinger) [inline watch.json]: is this on purpose?

Comment (ttseng) [reply watch.json]: yeah per this conversation: https://fogcreek.slack.com/archives/CJ1G77E87/p1575562388019200

Comment (clottman) [reply watch.json]: We talked about in #pod-shared-components. Without this line, you have to run `refresh` to see any changes you make. I'm not sure why it was there in the first place.

Comment (sarahzinger) [reply watch.json]: cool cool

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.19.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.19.0) :rocket:

**Pull request merged**